std_msgs/Header header

# if robot received its assigned goal location
bool received_goal

# if robot arrived to goal
bool arrived

geometry_msgs/Point point